src/test/java/nextstep/subway/favorite/FavoriteAcceptanceTest.java

```diff
@@ -1,8 +1,185 @@
+        광교역 = StationAcceptanceTest.지하철역_등록되어_있음("광교역").as(StationResponse.class);
+
+        LineRequest lineRequest = new LineRequest("신분당선", "bg-red-600", 강남역.getId(), 광교역.getId(), 10);
+        LineAcceptanceTest.지하철_노선_등록되어_있음(lineRequest).as(LineResponse.class);
+    }
+
+    /**
+     * Given 지하철역 등록되어 있음
+     * And 지하철 노선 등록되어 있음
+     * And 지하철 노선에 지하철역 등록되어 있음
+     * And 회원 등록되어 있음
+     * And 로그인 되어있음
+     * when 즐겨찾기 생성 요청을 하면
+     * then 즐겨찾기가 생성된다.
+     */
+    @DisplayName("즐겨찾기 생성 요청을 한다.")
+    @Test
+    public void createFavorite() {
+        //when
+        ExtractableResponse<Response> 즐겨찾기_생성_응답 = 즐겨찾기_생성_요청(accessToken, 강남역.getId(), 광교역.getId());
+        //then
+        즐겨찾기_생성됨(즐겨찾기_생성_응답);
+    }
+
+    /**
+     * Given 지하철역 등록되어 있음
+     * And 지하철 노선 등록되어 있음
+     * And 지하철 노선에 지하철역 등록되어 있음
+     * And 회원 등록되어 있음
+     * And 로그인 되어있음
+     * And 즐겨찾기 생성 되어있음
+     * when 즐겨찾기 목록 조회 요청을 하면
+     * then 즐겨찾기 목록이 조회된다.
+     */
+    @DisplayName("즐겨찾기 목록 조회 요청을 한다.")
+    @Test
+    public void findFavorites() {
+        //given
+        즐겨찾기_생성_요청(accessToken, 강남역.getId(), 광교역.getId());
+        //when
+        ExtractableResponse<Response> 즐겨찾기_조회_요청 = 즐겨찾기_조회_요청(accessToken);
+        //then
+        즐겨찾기_정보_조회됨(즐겨찾기_조회_요청, 강남역, 광교역);
+    }
+
+    /**
+     * Given 지하철역 등록되어 있음
+     * And 지하철 노선 등록되어 있음
+     * And 지하철 노선에 지하철역 등록되어 있음
+     * And 회원 등록되어 있음
+     * And 로그인 되어있음
+     * And 즐겨찾기 생성 되어있음
+     * when 즐겨찾기 삭제 요청을 하면
+     * then 즐겨찾기가 삭제된다.
+     */
+    @DisplayName("즐겨찾기 목록 삭제 요청을 한다.")
+    @Test
+    public void deleteFavorite() {
+        //given
+        즐겨찾기_생성_요청(accessToken, 강남역.getId(), 광교역.getId());
+        //when
+        ExtractableResponse<Response> 즐겨찾기_삭제_요청 = 즐겨찾기_삭제_요청(accessToken, 1L);
+        //then
+        즐겨찾기_삭제됨(즐겨찾기_삭제_요청);
+    }
+
+
+    /**
+     * when 등록되어 있지 않은 역으로 즐겨찾기 생성 요청을 하면
+     * then 에러가 발생한다.
+     */
+    @DisplayName("등록되어 있지 않은 역으로 즐겨찾기 생성 요청 시 에러발생.")
+    @Test
+    public void createFavoriteWithNoExistStation() {
+        //when
+        ExtractableResponse<Response> 즐겨찾기_생성_응답 = 즐겨찾기_생성_요청(accessToken, 999l, 1000l);
+        //then
+        응답에러_확인(즐겨찾기_생성_응답,HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+
+
+
+    public static void 즐겨찾기_정보_조회됨(ExtractableResponse<Response> response, StationResponse source, StationResponse target) {
+        List<FavoriteResponse> favoriteResponses = response.jsonPath().getList(".", FavoriteResponse.class);
+
+        List<StationResponse> favoriteSources = favoriteResponses.stream().map(favoriteResponse -> favoriteResponse.getSource()).collect(Collectors.toList());
+
+        List<StationResponse> favoriteTargets = favoriteResponses.stream().map(favoriteResponse -> favoriteResponse.getTarget()).collect(Collectors.toList());
+
+        assertThat(favoriteSources).contains(source);
+        assertThat(favoriteTargets).contains(target);
```

Comment:
assetAll 을 사용해보면 어떨까요? 😄 
```suggestion
        assertAll(() -> assertThat(favoriteSources).contains(source),
                () -> assertThat(favoriteTargets).contains(target));
```

Reply:
몰랐던 건데, 이렇게 쓰니까 한번에 적용할 수 있어 편하네요! 수정하였습니다.